### PR TITLE
Replacing an instace of `new Buffer.from()`

### PR DESCRIPTION
**Pull request by tlhunter**
- I believe this was an oversight from the commit 59314034f2bb1a1d622023d8acfec64961629364.

### CODE REVIEW

File: encodings/sbcs-codec.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ function SBCSCodec(codecOptions, iconv) {
         codecOptions.chars = asciiString + codecOptions.chars;
     }
 
-    this.decodeBuf = new Buffer.from(codecOptions.chars, 'ucs2');
+    this.decodeBuf = Buffer.from(codecOptions.chars, 'ucs2');
     
     // Encoding buffer.
     var encodeBuf = new Buffer.alloc(65536, iconv.defaultCharSingleByte.charCodeAt(0));
```
Comment (ashtuchkin) [inline encodings/sbcs-codec.js]: should this line be fixed too?

Comment (tlhunter) [reply encodings/sbcs-codec.js]: Yes, good point.

**Comment by tlhunter** [comment]
I'm using the GitHub web UI to update the commit; feel free to squash before merging!

**Comment by ashtuchkin** [comment]
yep, will do, thanks!
--
Alexander Shtuchkin

**Comment by ChALkeR** [comment]
@ashtuchkin Yes, this was uninitentional and is a oversight on my side, sorry for that.

**Comment by ashtuchkin** [comment]
Awesome, thanks everyone!

This will be included in the next release.

**Comment by ChALkeR** [comment]
@ashtuchkin @tlhunter Note that to my knowledge this change does not affect the actual behavior.

**Pull request merged**